tests/common/dualtor/dual_tor_io.py

```diff
@@ -200,11 +206,20 @@ def generate_from_server_to_t1(self):
         tcp_dport = TCP_DST_PORT
         tcp_tx_packet = testutils.simple_tcp_packet(
                       eth_dst=self.vlan_mac,
+                      eth_src=eth_src,
                       ip_src=self.from_server_src_addr,
                       ip_dst=self.from_server_dst_addr,
                       tcp_dport=tcp_dport
                  )
 
+        logger.info("-"*20 + "Server to T1 packet" + "-"*20)
```

Comment:
Could we also log the source port we're sending the packet from?

Reply:
Sure. Added the log in both the directions. I am also adding more logs as part of the PR following this - which addresses the single server being tested issue.